.github/no-response.yaml

```diff
@@ -0,0 +1,13 @@
+# Configuration for probot-no-response - https://github.com/probot/no-response
+
+# Number of days of inactivity before an Issue is closed for lack of response
+daysUntilClose: 14
+# Label requiring a response
+responseRequiredLabel: more-info-needed
```

Comment:
We don't use such label at the moment. Should we start doing it in your opinion?

Reply:
that would be my suggestion -- I added the label to the project already